app/scripts/views/coppa/coppa-age-input.js

```diff
@@ -56,6 +56,16 @@ define(function (require, exports, module) {
       age.val(age.val().substr(0, 3));
     },
 
+    onKeyDown: function (event) {
+      // force interger input and submit on enter
+      var x = event.which;
```

Comment:
`x` is pretty generic and could mean anything, can you use a more descriptive variable name? `keyCode` or similar would work.